src/server/config.ts

```diff
@@ -266,6 +266,7 @@ export const ApiConfig: ApiDataRequestConfig = {
       !process.env.BFF_ERFPACHT_API_URL,
     headers: {
       'X-HERA-REQUESTORIGIN': 'MijnAmsterdam',
+      'X-API-KEY': process.env.BFF_ENABLEU_ERFPACHT_API_KEY,
```

Comment:
Volgens mij moet het dit attribuut zijn, heb ik ook voor EnableU/PowerBrowser zo geïmplementeerd.
```suggestion
      'apiKey': process.env.BFF_ENABLEU_ERFPACHT_API_KEY,
```